modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -363,6 +371,12 @@ export default class Ecdsa {
       rk
     );
 
+    // create paillier challenge proof based on the other signers challenge
+    const sigma = EcdsaPaillierProof.prove(
```

Comment:
Since sigma is the proof, let's rename to call it `paillerProof` please everywhere we can. Otherwise it's going to be impossible for others to understand who haven't worked on this.